Xamarin/SSW.Rewards/SSW.Rewards/Pages/ScanPage.xaml.cs

```diff
@@ -9,29 +9,35 @@ namespace SSW.Rewards.Pages
     [XamlCompilation(XamlCompilationOptions.Compile)]
     public partial class ScanPage : ContentPage
     {
+        public const string EnableScannerMessage = "EnableScanner";
```

Comment:
Should this be `private` ?

Reply:
No. It's referenced elsewhere. See https://github.com/SSWConsulting/SSW.Rewards/issues/312 for resolving this.